### PR TITLE
fix: remove jexl logic and hide toggle

### PR DESCRIPTION
**Pull request by fkm**

### CODE REVIEW

File: addon/templates/components/cfb-form-editor/question.hbs [hunk 1]
```diff
@@ -76,7 +76,7 @@
           {{/if}}
         </div>
 
-        {{#if (not-eq f.model.__typename "StaticQuestion")}}
+        {{#if (and (or (eq f.model.isRequired "true") (eq f.model.isRequired "false")) (not-eq f.model.__typename "StaticQuestion"))}}
```
Comment (czosel) [inline addon/templates/components/cfb-form-editor/question.hbs]: I'd extract this into the `js` file.

Comment (fkm) [reply addon/templates/components/cfb-form-editor/question.hbs]: I agree in principle. But as all the other display logic is in the template, I did so as well.

Comment (czosel) [reply addon/templates/components/cfb-form-editor/question.hbs]: Yeah, I think the "soft limit" for this kind of logic should be 2-3 assertions. Would you mind extracting it?

Comment (fkm) [reply addon/templates/components/cfb-form-editor/question.hbs]: "Soft limit"?

Anyways. Is there a good way to observe a Changeset in a computed property?

Comment (czosel) [reply addon/templates/components/cfb-form-editor/question.hbs]: Looks like you found out :+1:

Comment (fkm) [reply addon/templates/components/cfb-form-editor/question.hbs]: Nope. This doesn't work.

Comment (czosel) [reply addon/templates/components/cfb-form-editor/question.hbs]: maybe @anehx can help?

File: addon/components/cfb-form-editor/question.js [hunk 2]
```diff
@@ -189,6 +189,13 @@ export default Component.extend(ComponentQueryManager, {
     return namespace ? `${namespace}-` : "";
   }),
 
+  requiredIsVisible: computed("changeset.{__typename,isRequired}", function() {
+    return (
+      (this.model.isRequired === "true" || this.model.isRequired === "false") &&
```
Comment (czosel) [inline addon/components/cfb-form-editor/question.js]: Shouldn't this be `this.changeset`?

Comment (fkm) [reply addon/components/cfb-form-editor/question.js]: Indeed. And it solves the problem.

The `this.model` was from my first attempt and I didn't test but went directly to Google. Thank you!

**Pull request merged**